ux.symfony.com/templates/components/TabbedCodeBlocks.html.twig

```diff
@@ -1,26 +1,26 @@
 <div {{ attributes.defaults({
     class: 'position-relative'
 }) }}>
-    <div class="terminal-code">
-        <ul class="nav nav-tabs">
-            {# set this in the child template #}
-            {% for filename in files %}
-                <li class="nav-item" role="presentation">
-                  <button class="nav-link {{ loop.first ? 'active' }}" data-bs-toggle="tab" data-bs-target="#{{ this.getItemId(filename) }}" type="button" role="tab" aria-controls="{{ this.getItemId(filename) }}" aria-selected="{{ loop.first ? 'true' : 'false' }}">{{ filename }}</button>
-                </li>
-            {% endfor %}
-        </ul>
-        <div class="tab-content" >
+    <div class="Terminal">
+        <div class="Terminal_header pb-0">
+            <ul class="nav nav-tabs">
+                {# set this in the child template #}
+                {% for filename in files %}
+                    <li class="nav-item" role="presentation">
+                        <button class="nav-link {{ loop.first ? 'active' }}"
+                                data-bs-toggle="tab" data-bs-target="#{{ this.getItemId(filename) }}"
+                                type="button" role="tab"
+                                aria-controls="{{ this.getItemId(filename) }}"
+                                aria-selected="{{ loop.first ? 'true' : 'false' }}"
```

Comment:
If we're going to use multiple lines for attributes, I'd rather break the `class` onto the next line, then indent everything only 4 spaces (instead of the 8 currently)